src/states.jl

```diff
@@ -96,9 +96,15 @@ struct PhaseDry{FT, PS} <: ThermodynamicState{FT}
     e_int::FT
     "density of dry air"
     ρ::FT
+    "air pressure"
+    p::FT
```

Comment:
Is this needed? I don't think it is.

Reply:
but the call or 'air_pressure' in relations/jl line 119:
air_pressure(ts::ThermodynamicState) = ts.p
is for all Phases. Without this it will break in tests